packages/plugins/response-cache/src/plugin.ts

```diff
@@ -354,7 +355,7 @@ export function useResponseCache<PluginContext extends Record<string, any> = {}>
           if (cacheControlDirective) {
             const cacheControlAnnotations = getDirective(schema, fieldConfig, 'cacheControl');
             cacheControlAnnotations?.forEach(cacheControl => {
-              const ttl = cacheControl.maxAge * 1000;
+              const ttl = cacheControl.maxAge != null ? cacheControl.maxAge * 1000 : null;
               if (ttl != null) {
                 ttlPerSchemaCoordinate[schemaCoordinates] = ttl;
               }
```

Comment:
Same here, we probably don't need the local variable :-)
```suggestion
              if (cacheControl.maxAge != null) {
                ttlPerSchemaCoordinate[schemaCoordinates] = cacheControl.maxAge * 1000;
              }
```

Reply:
Fixed, please mark as resolved if you are happy.